site/static/md/guide/datastar_expressions.md

```diff
@@ -47,6 +47,25 @@ The following example is valid because both `$foo.bar` and `$baz` are signals.
 
 Note that `data-*` attributes are evaluated in the order they appear in the DOM, so the `data-text` attributes must come _after_ the `data-signals-*` attributes in the example above. See the [attribute plugins reference](/reference/attribute_plugins) for more information.
 
+Multiple statements can be used in a single expression by separating them with a semicolon.
+
+```html
+<div data-signals-foo="1">
+  <div data-on-click="$foo++; @post('/endpoint')"></div>
```

Comment:
Perhaps these code snippets should use a `button` element instead of a `div` for the "on click" handlers so that the guide is demonstrating _accessible_ HTML code by default?

Reply:
Done, thanks!